### ISSUE TITLE
only trying to build ruby or python doesn't work via `remove java` - Execution failed for task ':compileJava'.

### ISSUE THREAD

**Issue by YiChengRepo** [google/protobuf-gradle-plugin#352]
Hi I have a project where I just wanted to build for ruby or python, after some study it seems have been suggested the right way of doing that is via `remove java` bcos by default the plugin adds it:

```
 task.builtins {
                remove java
                ruby {}
            }
```

But I got the following error when trying to run the gradle build:

```
> Task :compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> no source files
```

The full build.gradle is below 

```
plugins {
    id "com.google.protobuf" version "0.8.10"
    id 'java'
}

def grpcVersion = '1.24.0' // CURRENT_GRPC_VERSION
def protobufVersion = '3.9.0'
def protocVersion = protobufVersion
sourceCompatibility = 1.8
targetCompatibility = 1.8

group 'com.yi.chang.demo'
version '2.0-SNAPSHOT'

repositories {
    mavenCentral()
}

dependencies {
    testCompile group: 'junit', name: 'junit', version: '4.12'
    implementation "io.grpc:grpc-protobuf:${grpcVersion}"
    implementation "io.grpc:grpc-stub:${grpcVersion}"
    implementation "io.grpc:grpc-services:${grpcVersion}"
    compileOnly "javax.annotation:javax.annotation-api:1.2"
}

protobuf {
    protoc {
        artifact = 'com.google.protobuf:protoc:3.4.0'
    }
    plugins {
    }

    generateProtoTasks {
        // all() returns the collection of all protoc tasks
        all().each { task ->
            task.builtins {
                remove java
                ruby {}
            }
        }
    }
}


```

any input would be appreciated

**Comment by voidzcy** [comment]
Related to #333.

**Comment by voidzcy** [comment]
Same as #333, this should have been fixed by #375. Stay tuned for 0.8.14 release.
